src/requested-data/requested-data-state.ts

```diff
@@ -7,6 +8,7 @@ export interface RequestedDataState {
     votes: Votes[];
     metrics: Metrics[];
     parameters: Parameters[];
+    partyMap: Dictionary<string>
```

Comment:
This requires at the very least a minor bump in versioning if I recall correctly.

Reply:
Yup.